lib/router/index.d.ts

```diff
@@ -278,6 +278,8 @@ declare namespace createRouter {
 
         use(...handlers: HandlerArgument[]): this;
         use(mountPoint: string, ...handlers: HandlerArgument[]): this;
+        use(mountPoint: RegExp, ...handlers: HandlerArgument[]): this;
+        use(mountPoint: (string | RegExp)[], ...handlers: HandlerArgument[]): this;
```

Comment:
Is there any reason these signatures can't be `mountPoint: string | RegExp | (string | RegExp)[]`?